Support/Examples/OptionalFieldTest.kdl

```diff
@@ -0,0 +1,23 @@
+@type MacintoshPicture : "PICT" {
+    template {
+        HEXD Data;
+    };
+
+    field("PNG") {
+        Data as File<PNG> __conversion($InputFormat, PICT);
+    };
+
+    field("TGA") {
+        Data as File<TGA> __conversion($InputFormat, PICT);
+    };
+};
+
+declare MacintoshPicture {
+    new(#128) {
+        TGA = import "@rpath/TestImage.tga";
+    };
+
+    new(#129) {
+        PNG = import "@rpath/TestImage.png";
+    };
+};
```

Comment:
Is this optional field test supposed to be in here? There doesn't seem to be any code to support it currently?

Reply:
Disregard this, I noticed the comment on the issue.